account_banking_payment_export/models/payment_mode.py

```diff
@@ -43,9 +43,14 @@ def suitable_bank_types(self, cr, uid, payment_mode_id=None, context=None):
             res = [t.code for t in payment_mode.type.suitable_bank_types]
         return res
 
+    def _default_type(self):
+        return self.env.ref('account_banking_payment_export.'
+                            'manual_bank_tranfer')
+
     type = fields.Many2one(
```

Comment:
The method should return a recordset.. (even if it is empty)
It should work with the following method
```python
@api.model
def _default_type(self):
    return self.env.ref(
        'account_banking_payment_export.'
        'manual_bank_tranfer', raise_if_not_found=False)\
        or self.env['payment.mode.type']
```
Odoo will try to apply the default value a fist time during the installation step and give a warning. It will retry again during the post-install step and this time it should work.